src/main/kotlin/com/rootsid/wal/library/mongoimpl/BlockchainTxLogDocStorage.kt

```diff
@@ -0,0 +1,57 @@
+package com.rootsid.wal.library.mongoimpl
+
+import com.mongodb.client.MongoCollection
+import com.mongodb.client.MongoDatabase
+import com.rootsid.wal.library.mongoimpl.config.DefaultMongoDbConn
+import com.rootsid.wal.library.mongoimpl.document.BlockchainTxLogDocument
+import com.rootsid.wal.library.wallet.model.BlockchainTxAction
+import com.rootsid.wal.library.wallet.model.BlockchainTxLog
+import com.rootsid.wal.library.wallet.storage.BlockchainTxLogStorage
+import org.litote.kmongo.*
+import java.util.*
+
+class BlockchainTxLogDocStorage(db: MongoDatabase? = null, collectionName: String = "txLog") : BlockchainTxLogStorage {
```

Comment:
change the name of the collection to plural, like tx_logs

Reply:
ok